src/helm/benchmark/run_specs.py

```diff
@@ -2200,9 +2200,21 @@ def alter_run_spec(run_spec: RunSpec) -> RunSpec:
         if ANTHROPIC_MODEL_TAG in model.tags:
             add_to_stop_expander = AddToStopRunExpander(anthropic.HUMAN_PROMPT)
             increase_max_tokens_expander = IncreaseMaxTokensRunExpander(value=AnthropicClient.ADDITIONAL_TOKENS)
-            format_expander = FormatPromptRunExpander(
-                prefix=anthropic.HUMAN_PROMPT, suffix=f"{anthropic.AI_PROMPT} {AnthropicClient.PROMPT_ANSWER_START}"
-            )
+            # Get scenario tags
+            components = run_spec.scenario_spec.class_name.split(".")
+            module: Any = __import__(components[0])
+            for component in components[1:]:
+                module = getattr(module, component)
+            scenario_tags: List[str] = module.tags
```

Comment:
This seems pretty hacky so let's try to avoid this.  I think we shouldn't have PROMPT_ANSWER_START either.

I think for the new instructions, we should simply define the prompts properly so that the answer is well-formatted.

Reply:
@JosselinSomervilleRoberts @yifanmai Can we remove PROMPT_ANSWER_START or at least avoid applying it by default?